### PR TITLE
Change rtfd to readthedocs

### PR DESCRIPTION
**Pull request by pranay414**
This will fix issue: #5870

### CODE REVIEW

File: docs/automatic-redirects.rst [hunk 1]
```diff
@@ -20,10 +20,10 @@ The reasoning behind this is that RTD organizes the URLs for docs so that multip
 
 Among all the multiple versions of docs, you can choose which is the "default" version for RTD to display, which usually corresponds to the git branch of the most recent official release from your project.
 
-rtfd.org
+readthedocs.org
```
Comment (stsewd) [inline docs/automatic-redirects.rst]: This shouldn't be updated

File: docs/contribute.rst [hunk 2]
```diff
@@ -16,8 +16,8 @@ Get in touch
 - On IRC find us at `#readthedocs`_.
 
 .. _StackOverFlow: https://stackoverflow.com/questions/tagged/read-the-docs
-.. _on GitHub: https://github.com/rtfd/readthedocs.org
-.. _Gitter: https://gitter.im/rtfd/readthedocs.org
+.. _on GitHub: https://github.com/readthedocs/readthedocs.org
+.. _Gitter: https://gitter.im/readthedocs/readthedocs.org
```
Comment (saadmk11) [inline docs/contribute.rst]: I don't think `Gitter` should be changed.

Comment (stsewd) [reply docs/contribute.rst]: https://gitter.im/readthedocs/readthedocs.org yeah, looks like gitter doesn't get updated :/

File: docs/automatic-redirects.rst (hunk 1 above)
Comment (pranay414) [reply docs/automatic-redirects.rst]: What are PO files here?

Comment (stsewd) [reply docs/automatic-redirects.rst]: These https://github.com/readthedocs/readthedocs.org/pull/5871/files#diff-416b5b299618cc307035d37fc0899f2e

And, not, those should live in repo, but not modified in this PR

Comment (pranay414) [reply docs/automatic-redirects.rst]: @stsewd I've made the changes, can you check now?

File: docs/automatic-redirects.rst [hunk 3]
```diff
@@ -23,7 +23,7 @@ Among all the multiple versions of docs, you can choose which is the "default" v
 rtfd.org
 ~~~~~~~~
 
-Links to rtfd.org are treated the same way as above.
+Links to readthedocs.org are treated the same way as above.
```
Comment (stsewd) [inline docs/automatic-redirects.rst]: This shouldn't be changed

File: docs/custom_domains.rst [hunk 4]
```diff
@@ -117,7 +117,7 @@ An example nginx configuration for pip would look like:
         }
     }
 
-rtfd.org
+readthedocs.org
```
Comment (stsewd) [inline docs/custom_domains.rst]: this should remain unchanged

File: docs/custom_domains.rst [hunk 5]
```diff
@@ -117,7 +117,7 @@ An example nginx configuration for pip would look like:
...
 ---------
 
-You can also use `rtfd.io` and `rtfd.org` for short URLs for Read the Docs. For example, https://pip.rtfd.io redirects to its documentation page. Any use of `rtfd.io` or `rtfd.org` will simply be redirected to `readthedocs.io`.
+You can also use `readthedocs.io` and `readthedocs.org` for short URLs for Read the Docs. For example, https://pip.readthedocs.io redirects to its documentation page. Any use of `readthedocs.io` or `readthedocs.org` will simply be redirected to `readthedocs.io`.
```
Comment (stsewd) [inline docs/custom_domains.rst]: all these too, those are a domain, not the organization name in github

**Comment by pranay414** [comment]
@stsewd can you list down what all needs to be changed? The issue description seems vague to me.

**Comment by stsewd** [comment]
You should revert the changes in all .po files, and also revert the changes on domain names (rtfd.org)

**Comment by pranay414** [comment]
@stsewd I've made the required changes. Please review.

**Pull request merged**